java/src/test/java/com/saucelabs/saucebindings/SauceVisualSession.java

```diff
@@ -0,0 +1,66 @@
+package com.saucelabs.saucebindings;
+
+import com.saucelabs.saucebindings.visual.SauceVisualOptions;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.var;
+import org.openqa.selenium.InvalidArgumentException;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.MutableCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Map;
+
+public class SauceVisualSession {
+    private SauceVisualOptions visualOptions;
+    private RemoteWebDriver driver;
+
+    public SauceVisualSession() {
+    }
+
+    public RemoteWebDriver start() {
+        visualOptions = new SauceVisualOptions();
+//        visualOptions.setCapability("projectName", "visual-e2e-test");
+//        visualOptions.setCapability("viewportSize", "1280x1024");
+
+        SauceOptions sauceOptions = new SauceOptions();
+        sauceOptions.setVisualTesting(visualOptions);
+
+        driver = createRemoteWebDriver(getHubUrl(), sauceOptions.toCapabilities());
+        return driver;
+    }
+
+    protected RemoteWebDriver createRemoteWebDriver(URL url, MutableCapabilities capabilities) {
+        return new RemoteWebDriver(url, capabilities);
+    }
+
+    @Getter
+    @Setter
+    private String hubUrl = "https://hub.screener.io/wd/hub";
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/SauceLabs/sauce_bindings/pullRequest?prid=6239731)